### ISSUE TITLE
Enhancement request: Generate _category_.json/yml files for names of sidebar dropdown items

### ISSUE THREAD

**Issue by JenniBeadle** [sillsdev/docu-notion#17]
**Describe the issue**
Docusaurus uses the folder names (short with hyphens) for the sidebar dropdowns unless you add __category__.yml files to each folder to specify the name of the folder. These are also used to translate the folder names. 

See discussion on _Category item metadata_ in https://docusaurus.io/docs/sidebar/autogenerated#category-item-metadata

**Comment by hatton** [comment]
@JenniBeadle can you say how this will improve things for you?

(Especially if you're making an feature request, you'll want to remove boilerplate text like 

```
Describe the bug
A clear and concise description of what the bug is.
```

**Comment by JenniBeadle** [comment]
I guess you don't have folders in Notion so no wonder you didn't understood. I am trying to improve the folder names in the sidebar. 
  
A picture paints a thousands words. So let me show you what I want to reproduce as I 
have on my other sites.

**Here is what the sidebar looks like from Notion based Paratext-manual site:** 
<img width="217" alt="22 01_56_06" src="https://user-images.githubusercontent.com/108683165/197259810-13c4663b-ce7a-4acd-b4cc-45024a646676.png">
The text for the dropdowns is the folder name (complete with hyphens)

**Here is what the sidebar looks on my manual.paratext.org site when using edited _ _category_ _.json files**
<img width="145" alt="image" src="https://user-images.githubusercontent.com/108683165/197260036-bb7815cb-83c5-4c6e-96d2-14aae7c3a7aa.png">
The sidebar has clearer names (as supplied by the category files in each folder).

Also when I run the translate command, Docusaurus collects the folder names into a file (current.json) to translate.  I haven't worked with Crowd-In so I don't know how they do it. But here is what I got on the French locale of my other site:
<img width="145" alt="image" src="https://user-images.githubusercontent.com/108683165/197261106-9a8e7edd-fd39-418e-af5a-b9df5d8304ea.png">

When I had a repository on my computer I could just add the category files (even to the exported Notion). But with Release actions and no visible files, I can't to it. So  I need another way. Just changing the page in Notion doesn't work. 

On reflection I wonder if you can really export .json files from Notion. But there must be a way of achieving the end result.

**Comment by hatton** [comment]
The side bar items are named after the Outline pages. So instead of having your Notion Outline pages named in these shorthand ways like you are, they should be, based on your screenshots above:

|instead of| name it|
|---|---|
| Training Manual|Paratext Training Manual |
| Video Summaries| Paratext Video Summaries|
|Stage-1|, |Stage 1 Drafting|

Etc.

**Comment by JenniBeadle** [comment]
I had tried it before I did the enhancement request but it didn't work for me. **So what am I doing wrong?**

It definitely doesn't work from the cloned version in localhost because it is back in folders. 
So I tried the GitHub Release action and it changes spaces to "-" for categories. The file names are OK as it comes from the title.

Here is what I get 

<img width="219" alt="image" src="https://user-images.githubusercontent.com/108683165/197332896-fe3afe4c-2adb-4016-bd45-579e99367b89.png">

The ordering is out because I started to delete the initial numbering which normally controls the sort order. I thought it was supposed to be in same order as Notion. But maybe that was still an enhancement request.
